ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -62,6 +64,13 @@
   private static final int CHUNK_TIMEOUT_NS = 100 * 1000000; // 100ms
   private static final int MAX_CHUNK_SIZE_BETWEEN_FLUSHES = 8 * 1024; // 8K
 
+  /**
+  * To disable http2 support for okhttp3(js fetch used okhttp3),
+  * please put code below in your MainApplication.onCreate():
+  *      NetworkingModule.disableHTTP2 = true;
+  */
+  public  static boolean disableHTTP2 = false;
```

Comment:
Please pass this in using the constructor rather than using a global mutable variable. You should be able to override the module in `getNativeModules` of your app.

nit: extra space